server/server.js

```diff
@@ -20,9 +20,29 @@ server.listen(config.port);
 log.info('Server is on bruh and running on port: '+ config.port);
 
 //----------------------------------
+// Allow Cors Domain:
+//   res.header('Access-Control-Allow-Origin', 'MyDomainHere');
+//   This Allow for all client acces to your server
+//   Change for your IP or Domain or use localhost !
+//----------------------------------
+var allowCrossDomain = function(req, res, next) {
+    res.header('Access-Control-Allow-Origin', '*');
+    res.header('Access-Control-Allow-Methods', 'GET,PUT,POST,DELETE');
```

Comment:
Do we ever expect to have to allow `DELETE` endpoints? I don't think the player should ever have those exposed.